Sources/NetworkService/HTTPMethod.swift

```diff
@@ -0,0 +1,11 @@
+/// The HTTP request method.
```

Comment:
nit:

Again, take a look at [the RFC](https://datatracker.ietf.org/doc/html/rfc2616#section-5.1.1) and give more specific description:

```suggestion
/// The method to use in the HTTP request. Indicates the method to be performed on the resource identified by the Request-URI.
```

Reply:
Fixed